manage.py

```diff
@@ -1,11 +1,12 @@
 #!/usr/bin/env python
-import os
-import sys
+import os.environ.setdefault
+import sys.argv
 from django.core.management.commands.runserver import Command as runserver
+from cjapp.settings import DEFAULT_PORT
 
 if __name__ == "__main__":
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "cjapp.settings")
-    runserver.default_port = "8080"
+    runserver.DEFAULT_PORT = DEFAULT_PORT
```

Comment:
`DEFAULT_PORT` doesn't exist on runserver

```suggestion
    runserver.default_port = DEFAULT_PORT
```